meinberlin/templates/a4reports/emails/report_moderators.de.email

```diff
@@ -13,7 +13,7 @@
 in dem von Ihnen moderierten Projekt wurde ein Beitrag als unangemessen gemeldet.
 
 Gemeldeter Beitrag: <i style="color: #818181;">„{{ report.content_object }}“</i>,
-erstellt von {{ report.content_object.creator.username }} am {% language 'de' %} {{ report.content_object.created }} {% endlanguage %}
+erstellt von {{ report.content_object.creator.username }} am {% language 'de' %} {{ report.content_object.created|date:"d. F Y, H:i"}} Uhr.{% endlanguage %}
```

Comment:
The language de tag tells the date to display it in the German way. The date-filter, you added gives the date the specified format (you know, you added it because of that). As you don't need both, removing the language tag would be good

Reply:
Oh no, there was a weird language-problem with some emails, so not to end up with "23. May 2020, 13:12 Uhr" or something like that, leave the language tag in! Sorry!